src/Type/Definition/Type.php

```diff
@@ -261,4 +266,10 @@ public function jsonSerialize(): string
     {
         return $this->toString();
     }
+
+    public static function reset(): void
+    {
+        static::$standardTypes = [];
+        static::$builtInTypes = [];
```

Comment:
Also simpler with `unset`.

Reply:
It's not possible to unset static variables. I got that error.

Reply:
I do not like making `[]` a special case that means the variable is unset. Let's make them nullable and use `null` instead then.

Reply:
Good idea. Applied your feedback.